### PR TITLE
Hitbtc dev

### LINKED ISSUE #2: hitBTC API Scraping
We are looking for a Go developer that can provide scrapers of a websocket API. If our collaboration is productive we also offer tasks that include scraping of REST APIs. We reliably want to get trades in realtime from cryptocurrency exchanges. This trade information is then written into our databases using a specific format. We have an existing framework for this which allows us to parametrize the scraper. We work with Go and expect that the scrapers will work concurrently and efficiently.

We will provide the necessary documentation for the task and introduce you to our framework.    https://api.hitbtc.com

Set-up instruction for the task can be found under: http://docs.diadata.org/scraper.html
Documentation

### PR DESCRIPTION
**Pull request by PumpkingWok**
##### Description

- I have added the scraper for HitBTC exchange respecting the DIA documentation.
- I have developed it similar to CoinBaseScraper.
- I have insert only one crypto pair for now in `exchange-scrapers.json`.

##### Checklist

- [X] Changes don't break existing behavior.

##### Testing

- I have tested the code on Mac OS with go version 1.10.3.
- For using it i had to create a `/secrets` folder with json file for each exchange

##### Refers/Fixes

Fixes #2

### CODE REVIEW

File: exchange-scrapers/cmd/scraper.go [hunk 1]
```diff
@@ -66,6 +67,8 @@ func main() {
 					s[configPair.Exchange] = scrapers.NewCoinBaseScraper()
 				case dia.KrakenExchange:
 					s[configPair.Exchange] = scrapers.NewKrakenScraper(configApi.ApiKey, configApi.SecretKey)
+				case dia.HitBTCExchange:
+					s[configPair.Exchange] = scrapers.NewHitBTCScraper(configApi.ApiKey, configApi.SecretKey)
```
Comment (frranck) [inline exchange-scrapers/cmd/scraper.go]: Hi PumkpingWok, why using an api key and secrekey? you're not using it later.

File: exchange-scrapers/config/exchange-scrapers.json [hunk 2]
```diff
@@ -124,6 +124,11 @@
       "Symbol": "ZEC",
       "ForeignName": "ZECUSD",
       "Exchange": "Bitfinex"
+    },
+    {
+      "Symbol": "ETH",
+      "ForeignName" : "ETHBTC",
+      "Exchange" : "HitBTC"
     }
```
Comment (frranck) [inline exchange-scrapers/config/exchange-scrapers.json]: Could you please include the complete list of their pairs?

File: exchange-scrapers/scrapers/HitBTCScraper.go [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+			md := message.Params.(map[string]interface{})
+			ps, ok := s.pairScrapers[md["symbol"].(string)]
+
+			if ok {
+
+				md_data := md["data"].([]interface{})
+
+				for _, v := range md_data {
+
+					md_element := v.(map[string]interface{})
```
Comment (frranck) [inline exchange-scrapers/scrapers/HitBTCScraper.go]: Why not using a library like https://github.com/bitbandi/go-hitbtc ?

File: exchange-scrapers/cmd/scraper.go (hunk 1 above)
Comment (PumpkingWok) [reply exchange-scrapers/cmd/scraper.go]: Hi @frranck,
I have left these two parameters if in the future someone would implement a client other than connection to websocket, i can change this without problems.

File: exchange-scrapers/config/exchange-scrapers.json (hunk 2 above)
Comment (PumpkingWok) [reply exchange-scrapers/config/exchange-scrapers.json]: yes, i can add the complete list of the crypto pairs.I'm going to create a new commit in this PR.

File: exchange-scrapers/scrapers/HitBTCScraper.go (hunk 3 above)
Comment (PumpkingWok) [reply exchange-scrapers/scrapers/HitBTCScraper.go]: I have never seen it before, I have just looked in the repo and it seems to support only REST API, not WS.

File: exchange-scrapers/config/exchange-scrapers.json (hunk 2 above)
Comment (PumpkingWok) [reply exchange-scrapers/config/exchange-scrapers.json]: Hi @frranck,
I have just checked on HitBTC and there more than 500 pairs.Would you like that i include all pairs ? Thanks

Comment (frranck) [reply exchange-scrapers/config/exchange-scrapers.json]: could you please include the top 30 market caps?

File: exchange-scrapers/scrapers/HitBTCScraper.go (hunk 3 above)
Comment (frranck) [reply exchange-scrapers/scrapers/HitBTCScraper.go]: OK then let's keep your solution.

**Comment by frranck** [comment]
Thanks PumpkingWok !

**Pull request merged**